packages/vscode-extension/src/resources/Strings.json

```diff
@@ -107,7 +107,8 @@
     },
     "localDebug": {
       "portAlreadyInUse": "Port: %s is already in use. Close this port and try again.",
-      "portsAlreadyInUse": "Ports: %s are already in use. Close these ports and try again."
+      "portsAlreadyInUse": "Ports: %s are already in use. Close these ports and try again.",
+      "waitForBrowserStarted": "Please wait while the toolkit is starting your browser..."
```

Comment:
Why the verb `start` is past tense? What about `waitForBrowserToStart`?

Reply:
Nice catch, updated.